src/include/block.h

```diff
@@ -224,6 +229,7 @@ struct __wt_bm {
     void *map; /* Mapped region */
     size_t maplen;
     void *mapped_cookie;
+    bool is_remote; /* Whether the storage is located on a remote host */
```

Comment:
Is this going to be needed for anything? I don't mind it being in the PR but if it has future uses I'd like to have a rough idea of where we're going.

Reply:
I'm also curious why we need this.

Reply:
There are a few places where we would need to have different behavior based on whether we are using object storage. The only place I used it in this PR is in an assertion around mmap, as it's unsupported. Another use case that comes to mind is that we'd like to put in an assert for evicting prepared updates, but only when using the remote object storage. I can use a different name for this field, but we do need some way of checking this.

Reply:
Got it - thanks for the explanation.